README.md

```diff
@@ -48,7 +48,7 @@ DATABASES = {
 $ ./manage.py migrate
 $ ./manage.py createsuperuser
 
-$ ./manage.py ingest GMLFILE
+$ ./manage.py ingest GMLFILE 160229_130000
```

Comment:
... which does not respect that there is a time part.

> “Elements are perceived as part of a group if they are located within the same closed region.”

https://www.smashingmagazine.com/2014/03/design-principles-visual-perception-and-the-principles-of-gestalt/#common-regions